src/gui_tool/gui/sp_gui.py

```diff
@@ -55,33 +54,33 @@ def handle_keyboard(self, key_mapper: KeyMapper):
         sm = self.sm
         i = vcm.get_frame_index()
 
-        if key_mapper.consume("r"):
-            sm.reset()
-        elif key_mapper.consume("x"):
-            err = sm.split(i)
-            if err is not None:
-                self.error(err)
-            else:
-                self.log("New split made at {0}".format(i))
+        # if key_mapper.consume("r"):
```

Comment:
Do we want reset functionality to be removed right now?

Reply:
If you want to reset, just press escape twice.

That 'r' button is way too close to the 'e' button XD